cmd/utils.go

```diff
@@ -42,3 +49,212 @@ func getWorkerCount() int {
 	}
 	return nworkers
 }
+
+func GetPriorInstalledPackages(fileSystem afero.Fs, libraryPath string) map[string]desc.Desc {
+	installed := make(map[string]desc.Desc)
+	installedLibrary, err := fileSystem.Open(libraryPath)
+
+	if err != nil {
+		log.WithField("libraryPath", libraryPath).Fatal("package library not found at given library path")
```

Comment:
I believe you are correct, but I decided it was best to treat the code as if it would NOT crash, since log levels might be changed. In general, I don't like relying on a logging object to crash the program, but I guess I can see the logic there.

The important thing here is that control flow breaks -- I don't want someone in the future to change the log-level to "Error" and inadvertently cause the function to do weird things.

Thoughts?

Reply:
but the entire intent of Fatal is to explicitly cause everything to exit. If you _don't_ want that behavior then you drop to an error and return. 

You're intent:

> I believe you are correct, but I decided it was best to treat the code as if it would NOT crash, since log levels might be changed

actually is more confusing, as when reading the code, you say hmm Fatal should be exiting, but then there is a return after so the code is now telling me 2 different behavioral stories. Which is the expected behavior?